src/main/java/net/fabricmc/loom/configuration/providers/mappings/NoOpIntermediateMappingsProvider.java

```diff
@@ -29,19 +29,31 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 
+import org.gradle.api.Project;
+import org.gradle.api.provider.Property;
 import org.jetbrains.annotations.NotNull;
 
+import net.fabricmc.loom.LoomGradleExtension;
 import net.fabricmc.loom.api.mappings.intermediate.IntermediateMappingsProvider;
 
 /**
  * A bit of a hack, creates an empty intermediary mapping file to be used for mc versions without any intermediate mappings.
  */
 public abstract class NoOpIntermediateMappingsProvider extends IntermediateMappingsProvider {
 	private static final String HEADER = "tiny\t2\t0\tofficial\tintermediary";
+	private static final String HEADER_LEGACY = "tiny\t2\t0\tintermediary\tclientOfficial\tserverOfficial\t";
```

Comment:
Is Legacy the best name for this? Im not sure, I wonder if there is a better word to denote that the client/server had differing obf names. Legacy can mean a lot, and within loom its mostly used for anything pre 1.14.

Reply:
perhaps "obf merged" (for >=1.3) and "obf split" (for <1.3) for these sorts of things?